saleor/graphql/order/tests/test_order.py

```diff
@@ -2604,6 +2602,37 @@ def test_draft_order_delete(
         order.refresh_from_db()
 
 
+def test_draft_order_delete_non_draft_order(
+    staff_api_client, permission_manage_orders, order_with_lines
+):
+    order = order_with_lines
+    order.status = OrderStatus.UNFULFILLED
```

Comment:
We have more statuses different than `DRAFT` so how about to `parametrize` this test for all statuses to handle each cases?

Reply:
fixed